pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java

```diff
@@ -168,7 +168,7 @@ private double updateAvgResourceUsage(String broker, LocalBrokerData localBroker
                 brokerAvgResourceUsage.get(broker);
         double resourceUsage = localBrokerData.getMaxResourceUsageWithWeight(
                 conf.getLoadBalancerCPUResourceWeight(),
-                conf.getLoadBalancerMemoryResourceWeight(), conf.getLoadBalancerDirectMemoryResourceWeight(),
```

Comment:
Nit: it'd be helpful to document the `LoadBalancerMemoryResourceWeight` configuration to show that it is no longer in use and that it will be removed.

Reply:
The client module depends on the pulsar-common module, so the language level is still at 1.8, and we can not set `since` of Deprecated